README.Rmd

```diff
@@ -46,7 +46,7 @@ install.packages("xportr")
 ### Development version:
 
 ```{r, eval = FALSE}
-devtools::install_github("https://github.com/atorus-research/xportr.git", ref = "devel")
+install.packages("teal", repos = c("https://pharmaverse.r-universe.dev", getOption("repos")))
```

Comment:
Should 'teal' be updated to 'xportr' here? I'm not sure if 'xportr' is hosted on the pharmaverse.r-universe

Reply:
Oh sorry! You're absolutely right.

It is hosted there: https://pharmaverse.r-universe.dev/xportr